### PR TITLE
Add Kab locale

### PR DESCRIPTION
**Pull request by MohammedBelkacem**
Add Kab locale

### CODE REVIEW

File: app/assets/javascripts/components/locales/index.jsx [hunk 1]
```diff
@@ -22,6 +22,7 @@ import zh_hk from './zh-hk';
 import zh_cn from './zh-cn';
 import bg from './bg';
 import id from './id';
+import kab from './kab';
```
Comment (Artoria2e5) [inline app/assets/javascripts/components/locales/index.jsx]: Your `kab.jsx` seems missing from this PR. Try adding with `git add kab.jsx` before doing `git commit`.

**Review by Artoria2e5 — CHANGES_REQUESTED** [review]

File: app/assets/javascripts/components/locales/index.jsx (hunk 1 above)
Comment (belkacem77) [inline app/assets/javascripts/components/locales/index.jsx]: Thanks
The file is sent. The translation will be completed later.

**Comment by yookoala** [comment]
I couldn't find Kabyle language preference in neither Chrome or Firefox. That means the only way you could select the language is from the account preference screen. You'd need to at least update `app/helpers/settings_helper.rb` to have that option available. You'd probably need to update several JS, Ruby and YAML files to get it properly working.

Please read the documentations for more details:
https://github.com/tootsuite/documentation/blob/master/Contributing-to-Mastodon/Translating.md

File: app/assets/javascripts/components/locales/index.jsx (hunk 1 above)
Comment (BoFFire) [inline app/assets/javascripts/components/locales/index.jsx]: Hi @belkacem77 thank you very much for answering my e-mail :)
And welcome to kabyle language into Mastodon :)
If you need any help just ping me :)
Tanmirt ik.

**Comment by BoFFire** [comment]
@Artoria2e5 @yookoala I would like to notify you that @belkacem77 translated Mozilla Firefox (Thunderbird, VLC ...) into Kabyle language and many other software. Here is a web preview of this language, in case :) :
https://www.mozilla.org/kab/

**Comment by yookoala** [comment]
@BoFFire

Thanks. But I couldn't find Kab in the "Accepted-Language" selection: "Firefox > Preferences > Content > Languages". He might have missed that. Not trying to stop him from translating Kab. Just want to see that really in use.

Meanwhile, please provide the Kab translations.

**Comment by belkacem77** [comment]
Please you can check here: https://transvision.mozfr.org/?recherche=toolkit%2Fchrome%2Fglobal%2Fintl.properties%3Aintl.accept_languages&repo=central&sourcelocale=en-US&locale=kab&search_type=entities&entire_string=entire_string

If someone uses  Firefox kab locale,  you can also check here : http://www.reliply.org/tools/requestheaders.php

**Comment by yookoala** [comment]
The commit this PR based on is really old. The folder structure for javascripts has been vastly changed. I'd recommend you to redo your work on latest master and force push to your `patch-1` branch. Or you may finish you translation first. I can help do the relocating.

**Comment by Gargron** [comment]
Seems like this PR is missing the actual translation, so closing.

**Comment by belkacem77** [comment]
I will redo the work later. Thanks for help.

**Comment by BoFFire** [comment]
@mozillakab Update : kabyle available on https://crowdin.com/project/mastodon/kab#

**Comment by mozillakab** [comment]
Azul fell-ak.

Good thx

**Comment by BoFFire** [comment]
@mozillakab update https://github.com/tootsuite/mastodon/pull/12962

**Pull request closed without merging**